evento/app/controllers/events_controller.rb

```diff
@@ -19,6 +18,21 @@ def attendees
     render json: @event.attendees
   end
 
+  # POST /events/:id/attendees
+  def add_attendee
+    user = User.find(params[:user_id])
+    
+    if user != @current_user
+      render json: { error: 'Not Authorized' }, status: 401
+    elsif @event.attendees.include?(user)
+      render json: { message: 'You are already attendee of the event' }, status: 304
+    else
+      # success !
+      @event.attendees << user
+      @event.save # is this required? dunno. user.save? no clue
```

Comment:
Nahh but 

A: whats the worst it can do
B: model must be saved after modifying it. I don't see why it wouldn't bee the case here